app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    com, file_origin, cp_file = parts
+    if len(parts) == 3 and com == "cp" and file_origin != cp_file:
```

Comment:
you should check `if len(parts) == 3` before     `com, file_origin, cp_file = parts`